Geometry/MTDGeometryBuilder/interface/RectangularMTDTopology.h

```diff
@@ -67,18 +67,19 @@ class RectangularMTDTopology final : public PixelTopology
       m_yoffset = -(m_ncols + BIG_PIX_PER_ROC_Y*m_ncols/COLS_PER_ROC)/2. * 
 		  m_pitchy;
 
-      LogDebug("RectangularMTDTopology") << "nrows " << m_nrows << ", ncols " << m_ncols << ", pitchx "
-					   << m_pitchx << ", pitchy " << m_pitchy << ", xoffset "
-					   << m_xoffset << ", yoffset " << m_yoffset << ", BIG_PIX_PER_ROC_X "
-					   << BIG_PIX_PER_ROC_X << ", BIG_PIX_PER_ROC_Y " << BIG_PIX_PER_ROC_Y << ", ROWS_PER_ROC "
-					   << ROWS_PER_ROC << ", COLS_PER_ROC " << COLS_PER_ROC << ", ROCS_X " << ROCS_X << ", ROCS_Y " << ROCS_Y
-					   << "\nNROWS " << m_ROWS_PER_ROC * m_ROCS_X << ", NCOL " << m_COLS_PER_ROC * m_ROCS_Y;
+      LogDebug("RectangularMTDTopology") 
+      	<< "nrows " << m_nrows << ", ncols " << m_ncols << ", pitchx "
+	<< m_pitchx << ", pitchy " << m_pitchy << ", xoffset "
+	<< m_xoffset << ", yoffset " << m_yoffset << ", BIG_PIX_PER_ROC_X "
+	<< BIG_PIX_PER_ROC_X << ", BIG_PIX_PER_ROC_Y " << BIG_PIX_PER_ROC_Y << ", ROWS_PER_ROC "
+	<< ROWS_PER_ROC << ", COLS_PER_ROC " << COLS_PER_ROC << ", ROCS_X " << ROCS_X << ", ROCS_Y " << ROCS_Y
+	<< "\nNROWS " << m_ROWS_PER_ROC * m_ROCS_X << ", NCOL " << m_COLS_PER_ROC * m_ROCS_Y;      
     }
```

Comment:
There shouldn't be any implementation code in a header file, except for inlined code. It would be good to move all the implementation code to the .cc file, if you have time.

Reply:
I moved the implementation cose into the .cc file.